tests/odk_publish/etl/odk/test_qrcode.py

```diff
@@ -42,3 +42,21 @@ def test_create_app_user_qrcode(self, app_user):
         qr_code, collect_settings = create_app_user_qrcode(**kwargs)
         assert qr_code.getvalue()[:4] == b"\x89PNG"
         assert collect_settings == build_collect_settings(**kwargs)
+    
+    def test_create_app_user_qrcode(self, app_user):
```

Comment:
I think the CI failure is due to this test duplicating the prior test name.
```suggestion
    def test_create_app_user_with_admin_pw(self, app_user):
```